standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/metrics/MetricsConstants.java

```diff
@@ -61,6 +63,8 @@
   public static final String JVM_EXTRA_SLEEP = "jvm.pause.extraSleepTime";
 
   public static final String NUM_OPEN_TXNS = "num_open_transactions";
```

Comment:
+1 for leaving this here, TxnHandler uses this for its own calculations and I didn't want our changes to bother it:)